ExaFunctions/CsvBulkImportFromBlobStorage.cs

```diff
@@ -0,0 +1,96 @@
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Text;
+using Exasol.Utilities;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Exasol
+{
+    public static class CsvBulkImportFromBlobStorage
+    {
+        [FunctionName("CsvBulkImportFromBlobStorage")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                log.LogInformation("C# HTTP trigger function processed a request.");
+
+                string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+                dynamic data = RequestUtilities.DeserializeJsonBody(requestBody);
+
+                CheckParameters(data);
+                //POST body
+                //the target db connection string
+                string dbConnectionString = data?.dbconnectionstring;
+                //the target db table
+                string dbTable = data?.dbtable;
+                //the azure storage account connection string
+                string azureStorageAccountConnectionString = data?.storageaccountconnectionstring;
+                //the name of the blob storage container withing the storage account
+                string azureStorageAccountContainerName = data?.storageaccountcontainername;
+                //the filepath within the blob storage container
+                string azureStorageAccountContainerPath = data?.storageaccountcontainerpath;
+
+                var filenameList = AzureUtilities.GetAzureFilesListInBlobContainerFolder(azureStorageAccountConnectionString, azureStorageAccountContainerName);
+                var filteredFilenameList = FilterUtilities.FilterFilenameListOnDirectoryPath(filenameList, azureStorageAccountContainerPath);
+
+                string bulkloadingQuery = BulkloadingUtilities.ConstructBulkLoadingQuery(dbTable, azureStorageAccountConnectionString, azureStorageAccountContainerName, filteredFilenameList);
+                var response = QueryUtilities.RunNonQuery(dbConnectionString, bulkloadingQuery);
+                return new OkObjectResult(response);
+
+            }
+            catch (Exception e)
+            {
+                var errorObject = new JObject();
+                errorObject.Add("Something went wrong:", e.Message);
+
+                // Return a 400 bad request result to the client with JSON body
+                return new BadRequestObjectResult(errorObject);
+            }
+        }
+        private static void CheckParameters(dynamic data)
+        {
+            ////the target db connection string
+            if (data.dbconnectionstring == null)
+            {
+                throw new Exception("The database connection string was not included in the request and is required. Use \"dbconnectionstring\" to add this value to the request.");
+            }
```

Comment:
I'm not sure about the syntax of C#, but in Java I would extract methods here with names like validateTargetDBConnectionString and so on for each exception here. Then comments are not needed anymore and it would be easier to read

Reply:
It is 15 LoC including comments. 
If there were more I would've definitely have done so